scripts/helper/chron-ocs.sh

```diff
@@ -22,6 +22,17 @@ export LOGDIR="/home/test/logs/" #Dir for the logs.
 export LOGDATE=`date "+\%d\%H\%M"`
 export TESTUSR="test" # Test User
 
+# Check if another instance of the same script is running, if so quit.
+
+CHRONOCS="chron-ocs"
```

Comment:
This process is chron-ocs.  Should you be testing for "test-chron-ocs" which this script invokes below.  It is a long lived process as it waits for the ocs-upi-kvm task to complete.

Reply:
Yes, you are right, it is test-chron-ocs. This is how it is running in CI systems now.